internal/pkg/wwlog/wwlog.go

```diff
@@ -16,55 +16,39 @@ const (
 
 var (
 	logLevel = INFO
-	Indent   string
 )
 
 func SetLevel(level int) {
 	logLevel = level
 
-	if level == DEBUG {
-		log.SetFlags(log.LstdFlags | log.Lmicroseconds | log.Lmsgprefix)
-	} else {
-		log.SetFlags(0)
-	}
-
 	Printf(DEBUG, "Set log level to: %d\n", logLevel)
 }
 
-func SetIndent(i int) {
-	Indent = strings.Repeat(" ", i)
-}
-
-func prefixLevel(level int) {
+func prefixGen(level int) string {
```

Comment:
@gmkurtzer assuming that this gets call A LOT.    a golang "switch" on "level" would be guaranteed faster and more maintainable code over time

Reply:
Fixed, thanks for the suggestion!